Dockerfile

```diff
@@ -3,14 +3,18 @@ FROM python:3.7-slim
 ARG DEVELOPMENT
 
 # Install pip dependencies
-COPY . /kb/module
-RUN chmod -R a+rw /kb/module
+WORKDIR /kb/module
+COPY ./requirements.txt /kb/module
+COPY ./dev-requirements.txt /kb/module
```

Comment:
Pull out the `*requirements.txt` first so don't have to re-build image every time